pycbc/results/pygrb_plotting_utils.py

```diff
@@ -0,0 +1,640 @@
+# Copyright (C) 2019 Francesco Pannarale, Gino Contestabile
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+# =============================================================================
+# Preamble
+# =============================================================================
+
+"""
+Module to generate PyGRB figures: scatter plots and timeseries.
+"""
+
+import sys
+import os
+import logging
+import argparse
+import copy
+import numpy
+from pycbc.results import save_fig_with_metadata
+# TODO: imports to fix/remove
+try:
+    from glue import segments
+    from glue.ligolw import utils, lsctables, ligolw, table
+except ImportError:
+    MSG = "Fix when tests on top of the O2 PyGRB installation are finished."
+    logging.warning(MSG)
+    sys.exit()
```

Comment:
Remove the `sys.exit` here. I think that's the cause of at least 2 of the 3 travis failures. This should just raise a warning, then let the code fail normally if any of the routines using any of this code gets called.